raven/1.8.1/Dockerfile

```diff
@@ -0,0 +1,61 @@
+ARG RAVEN_VERSION=1.8.1
+
+# base image
+FROM ubuntu:jammy as app
+
+ARG RAVEN_VERSION
+
+# metadata
+LABEL base.image="ubuntu:jammy"
+LABEL dockerfile.version="1"
+LABEL software="raven"
+LABEL software.version="${RAVEN_VERSION}"
+LABEL description="Raven is a de novo genome assembler for long uncorrected reads."
+LABEL website="https://github.com/lbcb-sci/raven"
+LABEL license="https://github.com/lbcb-sci/raven/blob/master/LICENSE"
+LABEL maintainer="Erin Young"
+LABEL maintainer.email="eriny@utah.gov"
+
+# otherwise tzdata won't install
+ARG DEBIAN_FRONTEND=noninteractive
+
+# install dependencies via apt-get or yum if using a centos or fedora base 
+RUN apt-get update && apt-get install -y \
+    wget \
+    git \
```

Comment:
is `git` required? Can you remove it if not required

Reply:
That's a good question. Just a sec.

Reply:
Yes. It's required:

```
#10 [app 5/6] RUN cmake -S ./ -B ./build -DRAVEN_BUILD_EXE=1 -DCMAKE_BUILD_TYPE=Release -Wno-dev &&     cmake --build build &&     cmake --install ./build
#10 0.152 -- The CXX compiler identification is GNU 11.2.0
#10 0.161 -- Detecting CXX compiler ABI info
#10 0.251 -- Detecting CXX compiler ABI info - done
#10 0.262 -- Check for working CXX compiler: /usr/bin/c++ - skipped
#10 0.263 -- Detecting CXX compile features
#10 0.263 -- Detecting CXX compile features - done
#10 0.301 CMake Error at /usr/share/cmake-3.22/Modules/ExternalProject.cmake:2666 (message):
#10 0.301   error: could not find git for clone of bioparser-populate
#10 0.301 Call Stack (most recent call first):
#10 0.301   /usr/share/cmake-3.22/Modules/ExternalProject.cmake:3716 (_ep_add_download_command)
#10 0.301   CMakeLists.txt:23 (ExternalProject_Add)
```

Reply:
ah ok, no worries